pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -54,12 +55,13 @@ const (
 	customConfigVolumeName             = "custom-config"
 	customConfigMountPath              = "/var/lib/dynatrace/remotepluginmodule/secrets/config"
 	activeGateTrustedCertVolumeName    = "server-certs"
-	activeGateTrustedCertMountPath     = "/var/lib/dynatrace/secrets/ag"
+	activeGateTrustedCertMountPath     = "/var/lib/dynatrace/remotepluginmodule/secrets/ag"
```

Comment:
since you chnage path, i'm thinking to refactor and extrac basPath `/var/lib/dynatrace/remotepluginmodule/secrets/`. But it's okey if we can do it in seperate ticket after all.